R/bin_time.R

```diff
@@ -86,10 +100,12 @@
 #' #Assign randomly to overlapping bins based on fossil occurrence age range
 #' ex4 <- bin_time(occdf = occdf, bins = bins, method = "random", reps = 5)
 #'
-#' #Assign point estimates based on fossil occurrence age range
-#' ex5 <- bin_time(occdf = occdf, bins = bins, method = "point", reps = 5)
+#' #Assign point estimates following a normal distribution
+#' ex5 <- bin_time(occdf = occdf, bins = bins, method = "point", reps = 5,
+#'                 fun = dnorm, mean = 0.5, sd = 0.25)
 #' @export
-bin_time <- function(occdf, bins, method = "mid", reps = 100) {
+bin_time <- function(occdf, bins, method = "mid", reps = 100,
+                     fun = NULL, ...) {
```

Comment:
Should we supply a default for `fun`? Maybe `dunif` to preserve the original default behavior of the function?